client/idrepo/console/src/main/resources/org/apache/syncope/client/console/tasks/ExecutionsDirectoryPanel.properties

```diff
@@ -17,4 +17,5 @@
 start=Start
 end=End
 status=Status
+executor=Executor
```

Comment:
Please also add the new label to other languages' `ExecutionsDirectoryPanel_*.properties`, even with english value.
It's better to keep them all in sync.

Reply:
What would be the right translation for Italian?

Reply:
Esecutore